.changes/unreleased/tinymce-TINY-11756-2025-02-25.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added `allow_extra_mathml_attributes` and `allow_extra_mathml_elements` options for the editor.
```

Not sure we need to specify `editor` it's all for the editor also the Added is kind of implied by the category.

So what about:

```suggestion
body: New `allow_extra_mathml_attributes` and `allow_extra_mathml_elements` options.
```